src/components/layout/SendTip.vue

```diff
@@ -41,22 +44,29 @@
   import util from "../../utils/util";
   import aeternity from "../../utils/aeternity";
   import { EventBus } from "../../utils/eventBus";
+  import avatar from '../../assets/userAvatar.svg';
 
   export default {
     name: 'SendTip',
     components: {
       FiatValue
     },
     computed: {
-      ...mapGetters(['balance', 'loading'])
+      ...mapGetters(['balance', 'loading']),
+      isDisabled(){
+        return this.sendTipForm.amount <= 0
+                || this.sendTipForm.url.length === 0 
+                  || this.sendTipForm.url.indexOf('http') !== 0
```

Comment:
what does this check mean?

Reply:
I'm also wondering this.

Reply:
@stanislav-slavov can you clarify what this check does and maybe put more descriptive name to it?

Reply:
'does not start with ...'

Reply:
I have added this so we have a simple validation in place until there is a finished design for it. It disables the button, until we have url and some AE amount for a tip. I could update the name to a more descriptive one.

Reply:
validation should probably happen with a regex